pkg/sql/opt/xform/testdata/external/trading

```diff
@@ -1110,27 +1110,41 @@ limit
  │         ├── fd: (10)-->(15)
  │         ├── project
  │         │    ├── columns: quantity:14!null accountname:10!null
- │         │    ├── inner-join (hash)
+ │         │    ├── inner-join (lookup inventorydetails)
```

Comment:
This new query plan is better I think. Previously it was scanning `inventorydetails` (a large table) and only constraining by `accountname` (low cardinality). The new plan is a lookup join which is a "pseudo-scan" constraining  `accountname`, `dealerid`, and `cardid`. What do you think @andy-kimball?